src/ThemedStyleSheet.js

```diff
@@ -34,6 +72,7 @@ function get() {
 }
 
 function resolve(...styles) {
+  validateStyleInterface();
```

Comment:
create and resolve are pretty hot codepaths, so I'd like to avoid as much work as possible here. What do you think about wrapping all of these validate functions in something like

```js
if (process.env.NODE_ENV !== 'production') {
  ...
}
```

?

Reply:
I did some benchmarks of the change and didn't notice a difference between the typeof changes and the environment checks. If it matters, not sure on your environment support, but this would also assume/restrict the usage of this to a certain environment, without doing a null check on process and process.env. A babel plugin/webpack would probably simplify the check so it would definitely be better than not having it though.

What are your thoughts on an unimplemented function for the checks? It could get replaced on register, and we would no longer need to check each time create/resolve are called.  This would add some complexity to readability though. 

I would say the best two options in terms of performance are using the environment check you mentioned or the unimplemented function. What would you prefer?

Reply:
The environment checks can be compiled out by the app's build process, so it would be as if the code is deleted.